variant.go

```diff
@@ -58,9 +58,10 @@ type Variant struct {
 	Weight               *decimal.Decimal `json:"weight,omitempty"`
 	WeightUnit           string           `json:"weight_unit,omitempty"`
 	OldInventoryQuantity int              `json:"old_inventory_quantity,omitempty"`
-	RequireShipping      bool             `json:"requires_shipping,omitempty"`
+	RequireShipping      bool             `json:"requires_shipping"`
 	AdminGraphqlAPIID    string           `json:"admin_graphql_api_id,omitempty"`
 	Metafields           []Metafield      `json:"metafields,omitempty"`
+	Digital              Digital          `json:"digital,omitempty"`
```

Comment:
I can't find this in the docs - do you have a link where I can see how this is used?

Reply:
And wouldn't this need to be a `*Digital` for the `omitempty` to work?